service/models.py

```diff
@@ -131,6 +131,26 @@ def all(cls):
         return cls.query.all()
 
     @classmethod
+    def find_by_restock(cls, restock):
+        """ Returns all of the Inventory that quantity lower than their restock level
+        Args:
+            restock (boolean): if restock is true than return the list, if false than return normal list all
+        """
+        cls.logger.info('Processing quantity < restock_level query ...')
+        if restock is True:
+            return cls.query.filter(cls.quantity < cls.restock_level)
+        else:
+            return cls.query.all()
```

Comment:
Why we return all if restock == false?

Reply:
original I design restock==false as not setting restock, but I think you are right, quantity >= restock_level make more sense, fix it already